atc/metric/emitter/newrelic.go

```diff
@@ -85,37 +138,49 @@ func (emitter *NewRelicEmitter) simplePayload(logger lager.Logger, event metric.
 	return payload
 }
 
-func (emitter *NewRelicEmitter) emitPayload(logger lager.Logger, payload fullPayload) {
-	payloadJSON, err := json.Marshal(payload)
-	if err != nil {
-		logger.Error("failed-to-serialize-payload", err)
-		return
+func (emitter *batchEmitter) emitBatch(payload batchPayload) *loggableError {
+	var (
+		payloadReader io.Reader
+		err           error
+	)
+
+	payload = append(append([]byte{'['}, payload...), ']')
```

Comment:
is `payload` guaranteed to be a properly formatted json body that fits within a `[]` to form a proper json array?

I'm afraid this can lead to some trailing `,` problems 😬  

It seems weird to me to expact that `emitBatch` does the final wrapping of a properly formed payload, rather than just emitting it